release-schema.json

```diff
@@ -127,6 +144,16 @@
           "type": "string",
           "description": "A local identifier for this lot, such as a lot number. This is used in relatedLots references at the item, document and award level."
         },
+        "identifiers": {
+          "title": "Identifiers",
+          "description": "Identifiers for this lot. This field can be used to provide internal identifiers for the lot, such as identifiers from a buyer's document management system or procurement planning system.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SimpleIdentifier"
+          },
+          "uniqueItems": true,
+          "wholeListMerge": true
```

Comment:
`ocdskit schema-strict` didn't add `minItems`? Same for others. (Maybe it wasn't run because it added extra validation to existing fields - but new fields should conform to it.)

Reply:
I forgot to run it. Fixed in https://github.com/open-contracting-extensions/ocds_lots_extension/pull/41/commits/27825101d1fe15b86078bb4dc4dd78bb39e9521d.